central/debug/service/service.go

```diff
@@ -497,6 +498,8 @@ type debugDumpOptions struct {
 }
 
 func (s *serviceImpl) writeZippedDebugDump(ctx context.Context, w http.ResponseWriter, filename string, opts debugDumpOptions) {
+	subCtx, cancel := context.WithTimeout(ctx, debugDumpHardTimeout)
```

Comment:
Are you sure there is no timeout on context passed here?

Reply:
This will be called by [getDiagnosticDump](https://github.com/stackrox/stackrox/blob/master/central/debug/service/service.go#L676) / [getDebugDump](https://github.com/stackrox/stackrox/blob/e14bc41579f4b1ab5e47e62dad1182edc9400bd8/central/debug/service/service.go#L651) and we will pass the HTTP request's context there - this context won't have a deadline set.